src/library-authoring/collections/LibraryCollectionPage.test.tsx

```diff
@@ -37,6 +37,7 @@ const mockCollection = {
 };
 
 const { title } = mockGetCollectionMetadata.collectionData;
+jest.mock('../../content-tags-drawer/ContentTagsDrawer', () => jest.fn(() => <div>Mocked ContentTagsDrawer</div>));
```

Comment:
Could we at least test that the `ContentTagsDrawer` was initialized with the expected `id` collection usage key?

Reply:
Updated: https://github.com/openedx/frontend-app-authoring/pull/1379/commits/a01d7787311936908cb561fc648de725998abaaa